client/app/src/accounts/view/createSecondPassphraseConfirmation.html

```diff
@@ -0,0 +1,34 @@
+<md-dialog aria-label="Create Second Passphrase Confirmation" ng-cloak md-theme="ul.network.theme">
+  <form>
+    <md-toolbar>
+      <div class="md-toolbar-tools">
+        <h2><span translate>Second Passphrase Fee Alert</span></h2>
+        <span flex></span>
+        <md-button class="md-icon-button" ng-click="createSecondPassphraseConfirmationDialog.cancel()">
+          <md-icon md-font-library="material-icons">clear</md-icon>
+        </md-button>
+      </div>
+    </md-toolbar>
+    <md-dialog-content>
+      <div class="md-dialog-content">
+        <md-card md-colors="{'background': 'red-100'}" flex="100">
+          <md-card-title>
+            <md-card-title-text>
+              <span>
+                <translate><center>Second passphrase fee is <b>5 Ark</b>.  If you wish to continue, click NEXT.</center></translate>
```

Comment:
Can you make it read global constants rather than setting it hardcoded? We'll be changing fee structure soon.

Reply:
What's the best place to put that.. in app.js?  I see const in there already.  

In addition, I noticed that the in AccountService.js there is also hard-coded values for the second passphrase (in satoshis i.e. if (account.balance < 500000000) {...).  I can update this to globalize the values because not only is that value hard-coded, but in other areas as well. For example, extra delegate is 25 Ark but hard coded as if (account.balance < 2500000000) {

Reply:
I'm not coder just thinking out loud and looking for future releasing.

@luciorubeens What do you think?

Reply:
need to get fees from current peer, the api endpoint: https://node1.arknet.cloud/api/blocks/getfees

Reply:
just change `Ark` to current network token: `{{ul.network.token}}`

Reply:
@darfraider don't need a new dialog to confirm, just send the transaction to `validateTransaction` method in accountController

Reply:
validateTransaction only checks to ensure the user has enough Ark in their account.  But it doesn't warn them that they are about to spend that much right?  I've asked a few people that run 1.3.1 and they said the process went through and charged them but they had no clue it was going to be 5 ARK otherwise they would have not created the second passphrase.

Reply:
Nevermind -- I'll build the master branch because this must be new code (not in 1.3.1).  I'll look at leveraging the existing validateTransaction function.

Reply:
Why don't you just write the amount inside the create button:
CREATE (Ѧ5)      CANCEL